assets/model_monitoring/components/token_statistics/token_statistics_compute_metrics/spec.yaml

```diff
@@ -0,0 +1,49 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
+type: spark
+
+name: token_statistics_compute_metrics
+display_name:  Token Statistics - Compute Metrics
+description: Compute token statistics metrics.
+version: 0.0.1
+is_deterministic: true
+
+inputs:
+  token_dataset: 
+    type: mltable
+    mode: direct
+  group_pivot_column_name:
+    type: string
+    default: "user_id"
```

Comment:
Do we need this default value? If so, users have to explicitly set this to null to disable.